styles/stylesheet.css

```diff
@@ -0,0 +1,414 @@
+body {
+
+  margin:0;
+
+}
+
+div {
+  /*
+  min-height: 1000px;
+  margin-bottom: 100px;
+  */
+}
+
+header {
+  background-color: #2c9ab7;
+  background: #000000 url(../images/header-image.jpg) 100% 25% no-repeat;
+  margin:0;
+  padding:200px;
+}
+
+footer {
+
+ right: 0;
+ bottom: 0;
+ left: 0;
+ padding-top: 50px;
+ padding-bottom:50px;
+ background-color: #083b5f;
+ text-align: center;
+}
+
+section {
+
+  margin-left:255px;
+  margin-right:255px;
+  margin-top: 50px;
+  margin-bottom:50px;
+}
+
+.sectionblog {
+  padding-bottom:50px;
+}
+
+.mainimage {
+  float:left;
+  padding-right:50px;
+}
+.dogimage {
+  width:100%
+}
+
+.dogimage-small {
+  width:8%
+}
+.clearfix {
+  overflow: auto;
+  margin-left: 40px;
+  margin-right: 70px;
+}
+.clearfix-form {
+    overflow: auto;
+    margin-left: 20px;
+    margin-right: 300px;
+}
+.mainp {
+  font-family: 'Work Sans', sans-serif;
+  font-size: x-large;
+
+}
+.mainh1 {
+  margin-top: 0px;
+  font-size: xx-large;
+  font-family: 'Neuton', serif;
+  color: darkslategrey;
+
+}
+
+.dogsh1 {
+  font-size: xxx-large;
+  font-family: 'Neuton', serif;
+  color: darkslategrey;
+  margin-left: 40px;
+
+}
+.card {
+  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+  transition: 0.3s;
+  width: 20%;
+  margin-right:50px;
+}
+
+.small-card {
+  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+  transition: 0.3s;
+  width: 20%;
+}
+
+.card:hover {
+  box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2);
+}
+
+.container {
+  padding: 2px 16px;
+}
+
+.flex-container {
+  display: flex;
+  justify-content: center;
+
+}
+
+.headerp {
+  margin-top: .2rem;
+  margin-bottom: 0;
+  color: green;
+  padding: 0;
+  font-size:25px;
+  font-style: italic;
+
+}
+.cardp {
+  font-family: 'Work Sans', sans-serif;
+}
+.button {
+  width:100%;
+  background-color: #4CAF50; /* Green */
+  border: none;
+  color: white;
+  padding-top: 15px;
+  padding-right: 70px;
+  padding-left: 70px;
+  padding-bottom: 15px;
+  border-radius: 4px;
+  text-align: center;
+  font-weight:bold;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+  margin: 20px 2px;
+  font-weight:bold;
+  cursor: pointer;
+}
+
+.divheader {
+
+}
+.button-bottom {
+  width: 20%;
+      background-color: #ebebeb;
+      border: none;
+      color: darkslategrey;
+      padding-top: 15px;
```

Comment:
Styles look good for now, but you will need to update these to responsive units instead of px for the next assignment.